README.md

```diff
@@ -139,9 +152,26 @@ When using as a github app the flow is the following:
 
 1. Fork the ubiquibot repo and install the [ubiquibot-qa app](https://github.com/apps/ubiquibot-qa) on the forked repository.
 2. Enable github action running on the forked repo and allow `issues` on the settings tab.
-3. Create a [QA issue](https://github.com/ubiquibot/staging/issues/21) similar to this where you show the feature working in the forked repo
-4. Describe carefully the steps taken to get the feature working, this way our team can easily verify
-5. Link that QA issue to the pull request as indicated on the template before requesting a review
+3. ubiquibot-qa bot is a deployment of your fork's `development` branch. Make sure the changes you are testing are present in your fork's `development` branch.
+4. Create a [QA issue](https://github.com/ubiquibot/staging/issues/21) similar to this where you show the feature working in the forked repo
+5. Describe carefully the steps taken to get the feature working, this way our team can easily verify
+6. Link that QA issue to the pull request as indicated on the template before requesting a review
+
+## Payments Permits in a local instance
+
+For payment to work in your local instance, ubiquibot must be set up in a Github organization. It will not work for a ubiquibot instance set up in a personal account. Once, you have an ubiquibot instance working in an organization, follow the steps given below:
+
+1. Create a new private repository in your Github organization with name `ubiquibot-config`
+2. Add your ubiquibot app to `ubiquibot-config` repository.
+3. Create a file `.github/ubiquibot-config.yml` in it. Fill the file with contents from [this file](https://github.com/ubiquity/ubiquibot/blob/development/.github/ubiquibot-config.yml).
+4. Go to https://pay.ubq.fi/keygen and generate X25519 public/private key pair. Fill private key of your wallet's address in `PLAIN_TEXT` field and click `Encrypt`.
+5. Copy the `CIPHER_TEXT` and append it to your repo `ubiquibot-config/.github/ubiquibot-config.yml` as
+
+   `private-key-encrypted: "PASTE_YOUR_CIPHER_TEXT_HERE"`
+
+6. Copy the `x25519_PRIVATE_KEY` and append it in your local ubiquibot repository `.env` file as
+
+   `x25519_PRIVATE_KEY=PASTE_YOUR_x25519_PRIVATE_KEY_HERE`
```

Comment:
can you just change to capital letter X because env names are case-sensitive

Reply:
> Look good but I fear some new contributors might be confused because there are 2 ways to run the bot, as github action or locally.

I can fix this by adding following paragraph after the heading.


**You can perform a QA using your local instance of ubiquibot, or ubiquibot-qa app. The following steps describe how you can perform a QA using ubiquibot-qa.**


But do they need to change `X25519_PRIVATE_KEY` when they are using the QA bot? Encrypted private key is read from org config. And they don't have access to it, so they will not need to know this workflow key. What do you think?

Reply:
`X25519_PRIVATE_KEY` for github actions is here https://github.com/ubiquity/ubiquibot/blob/d56a523c81829e17513f7da1f1ccb1bbdca8b180/.github/workflows/bot.yml#L81

The problem I see with QAing with github actions is that they will want to change the workflow variables to use their own instance of supabase and logger...which will need a commit that will show up in the pull request so when they finish work they need to revert changes so we don't merge them to our development branch. 

I think we should only write instructions for running the bot locally because that's the easiest way to debug and QA, at least in my experience.

Reply:
I like this. Fewer complications at starting stage. I have updated docs to QA using local instances. I am marking it resolved, but feel free to mark it unresolved if something needs a change.